UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -838,9 +887,14 @@ public override string ToString()
         public class Tile : UndertaleObject, RoomObject, INotifyPropertyChanged
         {
             /// <summary>
-            /// Whether this tile is from an asset layer. Game Maker Studio: 2 exclusive.
+            /// Whether this tile is from an asset layer.<br/>
+            /// Equals true if it's Game Maker Studio: 2.
```

Comment:
```suggestion
            /// <see langword="true"/> for GameMaker Studio: 2 games, otherwise <see langword="false"/>.
```

Also, wonder if this attribute's setter should be limited. Since being able to just change it will very likely cause some issues.

Reply:
Regarding setter - `spriteMode` is public and has setter only for having an ability to make a copy of it, because room objects don't have `Clone()` method yet.

Reply:
Ah, ok.